.gitignore

```diff
@@ -1,3 +1,157 @@
-node_modules/
-dist/
+### WebStorm ###
```

Comment:
gotcha, that makes sense, can you add back in the line about "# Created by https://www.gitignore.io/api/webstorm,node" (which gives a bit more context/guides future users on how to edit it)